webapp/pages/faq.spec.js

```diff
@@ -0,0 +1,38 @@
+import { mount } from '@vue/test-utils'
+import Faq from './faq.vue'
+import VueMeta from 'vue-meta'
+
+const localVue = global.localVue
+localVue.use(VueMeta, { keyName: 'head' })
+
+describe('faq.vue', () => {
+  let wrapper
+  let mocks
+
+  beforeEach(() => {
+    mocks = {
+      $t: (t) => t,
+    }
+  })
+
+  describe('mount', () => {
+    const Wrapper = () => {
+      return mount(Faq, {
+        mocks,
+        localVue,
+      })
+    }
+
+    beforeEach(() => {
+      wrapper = Wrapper()
+    })
+
+    it('renders', () => {
+      expect(wrapper.is('div')).toBe(true)
```

Comment:
In general I like `.toBeTruthy()` instead `.toBe(true)`

Reply:
Sorry that I forgot about that @Mogge ! 😍